app/views/submissions/new.html.erb

```diff
@@ -1,4 +1,9 @@
-
+<%= content_tag :div, id: "submission-values", data: { t1_not_ready: Submission::NOT_READY,
+                                                          t5_more_than_2_5: Submission::MORE_THAN_2_5_GB,
+                                                          t5_more_than_10: Submission::MORE_THAN_10_GB,
+                                                          t6_not_agree: Submission::NOT_AGREE,
+                                                          t9_not_use: Submission::NOT_USE_CC0 } do %>
```

Comment:
@mikedaul You might be able to just self-close the content_tag without a do/end.